eth/catalyst/api.go

```diff
@@ -681,6 +682,9 @@ func (api *ConsensusAPI) delayPayloadImport(block *types.Block) (engine.PayloadS
 		// that the parent state is missing and the syncer rejected extending the
 		// current cycle with the new payload.
 		log.Warn("Ignoring payload with missing parent", "number", block.NumberU64(), "hash", block.Hash(), "parent", block.ParentHash(), "reason", err)
+		if strings.Contains(err.Error(), "forced head needed for startup") {
```

Comment:
Let's turn "forced head needed for startup" into a constant, so we can track where this error is being created and where it is being used.

Reply:
updated